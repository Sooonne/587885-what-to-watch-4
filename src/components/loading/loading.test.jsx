import React from "react";
import renderer from "react-test-renderer";
import Loading from "./loading";

it(`render Footer`, () => {
  const tree = renderer
  .create(<Loading
  />)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
