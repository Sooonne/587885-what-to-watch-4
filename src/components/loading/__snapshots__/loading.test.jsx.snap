// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Footer 1`] = `
<div
  className="user-page"
>
  <h1
    className="page-title user-page__title"
  >
    loading!
  </h1>
</div>
`;
